### PR TITLE
Update .plan

### LINKED ISSUE #89: Update november+december.plan
On order to have better overview what is missing, it would be good to have a up-to-date version of the `.plan` file.

### PR DESCRIPTION
**Pull request by spmse**
remove switch and variables from november+december.plan to keep the file clean
this partially fixes #89

### CODE REVIEW

**Review by dhonx — APPROVED** [review]

**Review by vbrazo — COMMENTED** [review]
We usually don't remove the lines. We just put a `+` symbol which means `done`.

File: november+december.plan [hunk 1]
```diff
@@ -3,8 +3,6 @@
 - example: json
 - example: testing
```
Comment (vbrazo) [inline november+december.plan]: `json` and `testing` are also done. Please update them :)

File: november+december.plan [hunk 2]
```diff
@@ -3,8 +3,6 @@
...
 - example: reading and writing files
-- example: switch
-- example: variables
 - example: mutable and immutable
 - example: operators (complete it)
 - example: pointers
```
Comment (vbrazo) [inline november+december.plan:6]: You can remove line `19` because GitBook is updating everything once we merge anything to the master branch.

**Comment by spmse** [comment]
Was not quite sure if I should delete elements or not, but in terms of consistency I did.

**Pull request merged**